### PR TITLE
Created simple web application that allows to register new accounts and create new characters

### PR DESCRIPTION
**Pull request by norzechowicz**

### CODE REVIEW

File: php/web/src/AppBundle/Entity/Character.php [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+
+    /**
+     * @var string
+     */
+    private $normalizedName;
+
+    /**
+     * @var int
+     */
+    private $health;
```
Comment (domis86) [inline php/web/src/AppBundle/Entity/Character.php]: add also: maxHealth

File: php/web/src/AppBundle/Entity/Character.php [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+    private $experience;
+
+    /**
+     * @param Uuid $accountId
+     * @param string $name
+     * @param int $health
+     */
+    public function __construct(Uuid $accountId, string $name, int $health)
+    {
+        $this->id = Uuid::uuid4();
```
Comment (domis86) [inline php/web/src/AppBundle/Entity/Character.php]: So here id of character is generated but also in: https://github.com/norzechowicz/no-game/blob/master/nodejs/server/src/NoGame/Engine/Player.js#L31
It should be generated in one place and in other places should be passed as param in constructor IMO

Comment (norzechowicz) [inline php/web/src/AppBundle/Entity/Character.php]: as you can see Player.js is still generated on the fly by Protocol.js - when I move to reading Players from storage I will remove generating id from Player.js

File: php/web/src/AppBundle/Entity/Character.php (hunk 1 above)
Comment (norzechowicz) [inline php/web/src/AppBundle/Entity/Character.php]: health is the maxHealth for the game, I will just add "currentHealth" later

Comment (domis86) [inline php/web/src/AppBundle/Entity/Character.php]: So also in Player.js lets make it the same: health+currentHealth

Comment (norzechowicz) [inline php/web/src/AppBundle/Entity/Character.php]: yea, Player.js will be adjusted sooner or later

**Comment by norzechowicz** [comment]
Few screenshots ;)

image

image

image

image

image

**Pull request merged**